components/builder-core/src/integrations.rs

```diff
@@ -1,132 +1,66 @@
-// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
+//! This module holds code that's common to dealing with the
```

Comment:
This isn't an issue with your PR necessarily, more a comment on the previous state of things: Naming things is hard, and in builder our integrations primitive is specific to cloud integrations like docker, aws, etc. I find myself wondering why our crypto code was in this module to begin with.

Reply:
Heh... I was wondering the same thing, and meant to just rename this, but forgot. Thanks for reminding me :joy: